Documentation/SRS.md

```diff
@@ -3,24 +3,15 @@
 ![image](https://user-images.githubusercontent.com/83345134/127284946-62fb3d35-98d4-40a6-8d0a-923d58bc3467.png "Add department")
 ![image](https://user-images.githubusercontent.com/83345134/127284991-63ca404a-6d7a-400a-a6e1-c7b6b9560a7e.png "Edit department")
 ![image](https://user-images.githubusercontent.com/83345134/127285071-2b699f96-0ac5-4421-b7ab-d78414e6d1d0.png "Delete department")
+![add error](https://user-images.githubusercontent.com/83345134/127475141-a99eb941-f19b-492b-979f-187fbaeca8a7.png)
+![edit error](https://user-images.githubusercontent.com/83345134/127475463-0334213d-f7ff-4065-9de9-25fd2f5c0fa4.png)
+![delete error](https://user-images.githubusercontent.com/83345134/127476504-d0bc1912-8c68-4eb9-8f93-77e5ee5a2c0f.png)
 
-![image](https://user-images.githubusercontent.com/83345134/127285373-3f14cb74-9510-4dc3-8e3a-2c2c01dee4a4.png "Edit employee")
-
-
-One of the site feature is managing employee resourses. This feature will take "employees" tab in main menu. The following tab requires big table for 
-describing every employee and button "add" - for adding new eployees.
-
-On the following picture situated eployee's tab:
 
+![image](https://user-images.githubusercontent.com/83345134/127285373-3f14cb74-9510-4dc3-8e3a-2c2c01dee4a4.png "Edit employee")
 ![image](https://user-images.githubusercontent.com/83345134/127285130-2928a7fa-dd6e-4b84-a077-ff80b723aa25.png "Employees list")
-
-
-By clicking green button user will add new employee. To add new employee user should write down his department, some personal data and his salary.
-Then user should press green button "save" to store this data in database.
-
 ![image](https://user-images.githubusercontent.com/83345134/127285174-8b015c12-a1b2-4ff9-a32c-d93a516a4bbb.png "Add employee")
-
-
-To delete an employee user should right-click eployee in the table and press delete button.
-
 ![image](https://user-images.githubusercontent.com/83345134/127285318-fc4847d6-ff49-4f51-863a-19ffafd29391.png "Delete employee")
+![add error](https://user-images.githubusercontent.com/83345134/127639007-3ceae0d9-212e-4714-8424-e515f535e0e3.png)
+![edit error](https://user-images.githubusercontent.com/83345134/127639014-e66fa490-4b0f-487c-8006-6faf4f215ced.png)
+![delete error](https://user-images.githubusercontent.com/83345134/127639021-fb06c230-6baa-466c-a253-d553aeb58a00.png)
```

Comment:
i need extra mockup: a validation error for invalid date on the display list of employees 
also as ihar said, there should be two validation errors for adding and edidting employees and three system errors for adding editing or deleting employees